v20.2/postgresql-compatibility.md

```diff
@@ -5,7 +5,7 @@ toc: true
 redirect_from: porting-postgres.html
 ---
 
-CockroachDB supports the PostgreSQL wire protocol and the majority of its syntax. This means that your existing applications can often be migrated to CockroachDB without changing application code.
+CockroachDB supports v3.0 of the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and the majority of PostgreSQL syntax. This means that existing applications built on PostgreSQL can often be migrated to CockroachDB without changing application code.
```

Comment:
`v13`

Reply:
The version of the protocol that we support is v3.

From https://www.postgresql.org/docs/current/protocol.html:

> https://www.postgresql.org/docs/current/protocol.html

Reply:
Sounds good, didn't know they didnt keep the same versioning

Reply:
> didn't know they didnt keep the same versioning 

Yeah IDK why they don't - just makes it more confusing. TFTR!